modules/platforms/cpp/core/src/impl/cluster/cluster_group_impl.cpp

```diff
@@ -568,13 +566,18 @@ namespace ignite
                 return *ReadNodes(reader).Get();
             }
 
-            long ClusterGroupImpl::GetTopologyVersion()
+            int64_t ClusterGroupImpl::GetTopologyVersion()
             {
                 RefreshNodes();
 
                 return static_cast<long>(topVer);
             }
 
+            SP_ClusterGroupImpl ClusterGroupImpl::GetEmptyClusterGroup()
+            {
+                return ForNodeId(Guid(0, 0));
```

Comment:
Looks like a hack. Not very critical, but I wonder, how the same thing done in .NET?